app/mailers/teacher_mailer.rb

```diff
@@ -31,6 +32,14 @@ def application_declined
     )
   end
 
+  def application_declined_following_further_information_request
```

Comment:
This name, not having it's own locale key for the subject, will break the timeline entry component:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/be8908619a78fd1f5998312ac893f5b00a3e0b53/app/components/timeline_entry/component.rb#L81-L86

I don't mind this approach, but I wonder if it would be better if we passed in `further_information: true` or something in the `params` instead of having an entirely new view, and then wrote some `erb` in the view to customise the content. It feels to me like these emails are similar enough that would be a better approach, and means content common to both of them would stay in sync.

Reply:
I'll sort the key out. I did think of the approach you're suggesting. I decided it was better to make it explicit that they're treated differently. I don't really mind either way though.